README.md

```diff
@@ -1,21 +1,60 @@
 
-We have a BETA version of our [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/) available to help you learn how to use the APIs. The Swagger UI currently only supports Adobe ID authentication.
+It is assumed that you have an understanding of the Adobe Analytics product, its features and capabilities and that you know how to use the product. Thus an introduction to Adobe Analytics is outside of the scope of this documentation. For more information about the Adobe Analytics product, refer to the [Adobe Analytics documentation](https://marketing.adobe.com/resources/help/en_US/analytics/getting-started/).
 
-* Getting Started
-  * [Creating an OAuth Client](create-oauth-client.md)
-  * [OAuth using cURL](oauth-curl.md) - Example OAuth authentication flow using cURL commands
-  * [OAuth using POSTMAN](oauth-postman.md)
-* [Reporting API Guide](reporting-guide.md)
-* [Migration Guide](migration-guide.md) - Help for users migrating from the 1.3/1.4 APIs
-* [Calculated Metrics](calculatedmetrics.md)
-* [Virtual Report Suites](virtualreportsuites.md)
+This documentation includes the following sections:
+
+1. [Getting Started](#getstart)
+
+1. [Swagger Endpoint Descriptions](#endpoints)
+
+1. [Reporting API Guide](#reporting)
+
+1. [Segments API Guide](#segments)
+
+1. [Using Calculated Metrics](#calcmet)
+
+1. [Creating Virtual Report Suites](#virtual)
+
+1. [Migrating from 1.3/1.4 APIs to 2.0 APIs](#migration)
+
+
+## <a name="getstart">Getting Started</a>
+To get started with Analytics 2.0 APIs:
+
+1. Create an [OAuth client on the Adobe I/O Console](create-oauth-client.md).
+
+2. Use [cURL for OAuth authentication](oauth-curl.md).
+
+OR
+
+2. Use [Postman for OAuth authentication](oauth-postman.md).
+
+
+## <a name="endpoints">Swagger Endpoint Descriptions</a>
+The Analytics 2.0 API endpoints and methods are described on our [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/). It currently only supports Adobe ID authentication.
+
+## <a name="reporting">Reporting API Guide</a>
+The [Reporting API Guide](reporting-guide.md) provides configuration guidance and best practices for the ```/reports``` endpoint.
+
+## <a name="segments">Segments API Guide</a>
+The [Segments API Guide](https://github.com/AdobeDocs/analytics-2.0-apis/blob/master/reporting-guide.md) provides configuration guidance and best practices for the ```/segments``` endpoint.
+
+
+## <a name="calcmet">Using Calculated Metrics</a>
+The following examples show how to retrieve calculated metrics information:
+
+## <a name="virtual">Using Calculated Metrics</a>
```

Comment:
Using Calculated Metrics is listed twice here

Reply:
Removed.